src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#if HAVE_GSSFW_HEADERS
+typedef struct gss_name_t_desc_struct GssName;
```

Comment:
I don't understand the typedefs here. Why can't you just forward declare these structs?